root/static/scripts/relationship-editor/components/RelationshipPhraseGroup.js

```diff
@@ -143,7 +143,11 @@ const RelationshipPhraseGroup = (React.memo<PropsT>(({
       [backward ? 'entity1' : 'entity0']: source,
       attributes: newAttributesData,
       linkOrder: maxLinkOrder > 0 ? (maxLinkOrder + 1) : 0,
-      linkTypeID: linkTypeId,
+      /*
+       * The `typeId` on `RelationshipLinkTypeGroupT` stores null types as 0
+       * for easier sorting. Convert them back to null here.
```

Comment:
To follow https://github.com/metabrainz/musicbrainz-server/pull/2868/files#r1118960502

```suggestion
       * The `typeId` on `RelationshipLinkTypeGroupT` stores null types as 0
       * (which isn’t a valid relationship type row id anyway)
       * for easier sorting. Convert them back to null here.
```

Reply:
Thanks, merged your suggestion in together with other small tweaks.